### ISSUE TITLE
Toggle measure 0 (anacrusis); allow incomplete measures

### ISSUE THREAD

**Issue by jasconias** [tscz/transcribe#33]

**Comment by tscz** [comment]
@jasconias: Good Point. However, I thought of the measure 0 of an option to model the musical anacrusis (see https://en.wikipedia.org/wiki/Anacrusis). Especially in typical pop songs which are based on sets of 2 or 4 measures, it makes it more intuative to count (i.e. measure 1-8 verse1, 9-16 chorus1, 17-25 verse2 ... instead of 2-9 verse1, 10-17 chorus1, 18-26 verse2, ...). Maybe it is more intuitive to be able to mark Measure1 as user, but still use Measure 0 as indicator for the "Auftakt". What do you think?

**Comment by jasconias** [comment]
Makes sense, didn't think of that. Though the "Auftakt" is usually not a full measure, so ideally there should be a way of indicating incomplete measures (maybe something to have in mind when implementing meter changes within songs, as this feels related).
